docs/about/index.md

```diff
@@ -27,3 +27,27 @@ Multiple [.NET implementations and products](products.md) are available, based o
+
+Characteristics
+---------------
+
+**Multi-Language** -- .NET provides a well-defined type system, formats, runtime and framework and tools that can be used by multiple languages, both for their own execution and also to interoperate with other languages using those same components of .NET as their shared curency.
+
+**Managed Memory** -- .NET manages automatically manages memory for you via a garbage collector. It ensures that you always reference live objects, guaranteeing that you avoid nasty problems like buffer overruns and access violations. This includes array bounds checking.
+
+**Type Safety** -- The primary .NET model for functionality and memory representation is "types". Types define shape and optionally behavior. The runtime ensures that calling code can only operate on types according to their defintion and specified visibility of members, providing consistent, reliable and secure results.
+
+**User-defined Value Types** -- Value types are a useful category of types since they offer the semantic of "pass by value" instead of "pass by reference", as is the case for classes. Value types are most obviously usefuly for numeric data. .NET enables value types for both primitives types, like integers, and user-defined types.
+
+**Generic types** -- Generic types are types with one or more type parameters that can be specified on a per-instantiation basis. This is useful for many types, which otherwise would  expose contents as the Object type or require multiple type definitions. For example, a given instantiation of a collection type can be made specific to People, GPS locations or strings.
+
+**Reflection** -- .NET defines a metadata format that describes the types within a binary. The reflection subsystem uses this data, exposing APIs for both reading and instaniating types at runtime. This facility is very useful for dynamic scenarios where it is not convenient to know the exact implementation of a program ahead of time.
+
+**Flexible code generation** -- .NET does not prescribe a specific approach to transforming .NET binaries into machine code. Many approaches have been used successfully, including interpretation, just-in-time (JIT) compilation, ahead-of-time (AOT) compilation with JIT fall-back and AOT with no JIT fall-back. Each of these strategies can be valuable and there are opportunities for using them together.
+
+**Cross-platform** -- .NET was intended to be cross platform from its inception. The binary format and the instruction set are operating system, CPU and pointer-size agnostic. A given .NET binary built in 2000 to run on a 32-bit Windows machine can run on the ARM64 iOS device on 2016 without issue.
+
+**Open source** -- The .NET Core and Mono implemenations of .NET are open source, using the MIT license. These general purpose runtimes can be used as the basis of academic research or teaching or commercial products. The open nature also means that anyone can contribute back to the upstream product code.
```

Comment:
typo: implemenations 
The open nature -> The open nature of the source (or any any other noun)?